WireDomain/Sources/WireDomain/Notifications/Actions/NotificationAction.swift

```diff
@@ -0,0 +1,62 @@
+//
+// Wire
+// Copyright (C) 2025 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import UserNotifications
+
+protocol NotificationAction {
```

Comment:
I noticed that all the conformers are structs with hardcoded values. For me I don't see the benefit of having a protocol, where a single `NotificationAction` struct with predefined static helpers would suffice:

```swift
struct NotificationAction {

  let identifier: String
  let title: String
  ...

}

extension NotificationAction {

  static let muteConversation = NotificationAction(
    identifier: "muteConversationNotificationAction,
    title: "Mute conversation",
    ...
  )

}

```

What do you think?